web/js/codeworld.js

```diff
@@ -585,9 +643,13 @@ function run(hash, dhash, msg, error) {
     if (hash) {
         window.location.hash = '#' + hash;
         document.getElementById('shareButton').style.display = '';
+        if (!!navigator.mediaDevices.getUserMedia) {
```

Comment:
Don't you want to remove the changes in this block?  They'll be undone by the code below, anyway.